pom.xml

```diff
@@ -63,6 +67,11 @@
             <artifactId>modelmapper</artifactId>
             <version>${modelMapper.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.sonarsource.scanner.maven</groupId>
+            <artifactId>sonar-maven-plugin</artifactId>
+            <version>3.8.0.2131</version>
```

Comment:
Can we move the version to properties section?

Reply:
Done.